rero/rero-ils-ui
# document: display title in brief and detail views

* Adds variant titles and responsibility in detail view.
* Updates titles in both views.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>
Co-Authored-by: Gianni Pante <gianni.pante@rero.ch>

## Why are you opening this PR?

To implement task 1360 of US1232
https://tree.taiga.io/project/rero21-reroils/task/1360?kanban-status=1224894

## How to test?

Needs https://github.com/rero/rero-ils/pull/840 (now merged in US1232 branch).
* Go to brief view of documents and check the display of the title (public and professional).
* In admin view, select `arabic` documents and check that `Title`, `Variant title` and `Statement of responsibility` are displayed in the detailed view, if they exist in the bibliographic record.
* Check other records.
Example: `http://localhost:4200/records/documents/detail/34` should display all fields.
* Check views such as item detailed view, list of requests, list of loans, fees, pickup, etc.

## Code review check list
